index.html

```diff
@@ -7,10 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+  <link rel="StyleSheet" href="css/style.css">
 </head>
 
+
 <body>
-    <!-- Add your markup here -->
+    <header>  
+        <img class="logo-image" src="Images/logo.jfif" alt="logo">
+        <p class="hamburger">&#9776</p>
```

Comment:
The hamburger menu isn't working 😞 This is the video I watched when I learned how to add one, it's a bit long though but really helpful https://www.youtube.com/watch?v=DZg6UfS5zYg